Cargo.lock

```diff
@@ -1866,13 +2262,37 @@ dependencies = [
  "zstd 0.13.0",
 ]
 
+[[package]]
+name = "datafusion-cli"
+version = "33.0.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "f51d09ed6f01e41a983cf720228b4f0811d755b3f52776dd5170c744bd5d626e"
+dependencies = [
+ "arrow",
+ "async-trait",
+ "aws-config 0.55.3",
```

Comment:
Also just noticed that we are compiling two versions of many of the `aws-*` dependencies.

Looks like `kamu` depends on `aws-* = 0.57` while `datafusion-cli` pulls in `aws-* = 0.55`.

Unfortunately it doesn't look like we can disable AWS as a feature in `datafusion-cli`:
https://github.com/apache/arrow-datafusion/blob/main/datafusion-cli/Cargo.toml

@Wizzy-wooz  - leave this problem to us. 

I think our options are:
- Accept duplication of dependencies for now
- Submit a patch to `datafusion-cli` to bump the version of `aws-*` libraries
- Submit a patch to make AWS feature optional

Reply:
Decided to copy the code over for reasons described in new crate's README.